resource/abstractions/individual_access.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Individual Access
+
+General Resource Profile Administration functions for checking an Individual User's access to a General Resource Profile. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_my_access()`
+
+```python
+def get_my_access(self, resource: str, class_name: str) -> Union[str, bytes, None]:
+```
+
+#### 📄 Description
+
+Get the **Running User's** access to a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The name of the **general resource profile** to evaluate with this command.
+
+* `class_name`<br>
+  The name of the **class** the specified general resource profile belongs to.
+
+#### 📤 Returns
+* `Union[str, bytes, None]`<br>
+  Returns `None` when the general resource profile has no **Your Access** section defined for the running userid, otherwise returns the access level as a string. If the `ResourceAdmin.generate_requests_only` class attribute is set to `True`, **concatenated Security Request XML bytes** will be returned.
```

Comment:
Change `otherwise returns the access level as a string.` to `otherwise the **Access Level** is returned as a string.`

Reply:
Changed in both places.